src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/arm/matmul.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "single_layer_tests/classes/matmul.hpp"
+#include "shared_test_classes/single_layer/mat_mul.hpp"
+#include "test_utils/cpu_test_utils.hpp"
+#include "test_utils/fusing_test_utils.hpp"
+
+using namespace InferenceEngine;
+using namespace CPUTestUtils;
+using namespace ngraph::helpers;
+using namespace ov::test;
+
+namespace CPULayerTestsDefinitions {
+namespace MatMul {
+namespace {
+#ifdef OV_CPU_WITH_MLAS
+std::vector<CPUSpecificParams> filterSpecificParams_MLAS() {
+    // replace with mlas primitive type
+    std::vector<CPUSpecificParams> specificParams;
+    specificParams.push_back(CPUSpecificParams{{}, {}, {"acl"}, "acl"});
+    return specificParams;
+}
+
+std::vector<fusingSpecificParams> fusingParamsSet3D_MLAS_smoke {
+        emptyFusingSpec,
+        fusingBias,
+        fusingMultiplyPerChannel
```

Comment:
emptyFusingSpec -> common
fusingBias -> x86
fusingMultiplyPerChannel -> x86

Reply:
addressed the commend by moving MLAS specific tests to `x64\mlas` directory
MLAS specific tests should not be executed on ARM since MLAS Matmul is not supported on ARM.